### ISSUE TITLE
Review for the journal 10-11-2016-do-me-a-favor

### ISSUE THREAD

**Issue by abruzzihraig** [abruzzihraig/road-for-revenge#72]
Any advice would be appreciated, thanks!

**Comment by InfinityBlue** [comment]
>  But she ignored the warning then worked harder like before

如果是 worked harder 的话，后面是 than 是不是比较好，或者就说worked hard？
我也不确定是不是有问题，只是看过去觉得有些古怪。

**Comment by abruzzihraig** [comment]
@InfinityBlue Yes, I think you are right. It should be either 'worked harder than before' or 'worked hard like before.' Thanks, bro!

**Comment by abruzzihraig** [comment]
I find the first sentence 'My girlfriend showed me a short video to me about a girl had a cancer.' has a grammar mistake.
It should be 'My girlfriend showed me a short video about a girl had cancer' or 'My girlfriend showed a short video to me about a girl had cancer'.

**Comment by InfinityBlue** [comment]
@abruzzihraig 对喔
